src/frontend/components/SøknadsSteg/Kvittering/Kvittering.tsx

```diff
@@ -81,6 +84,12 @@ const Kvittering: React.FC = () => {
                 </Normaltekst>
             </KomponentGruppe>
 
+            {!eøsSkruddAv && (
```

Comment:
Bare for å dobbeltsjekke at dette blir riktig, skal denne teksten alltid vises etter at vi går live med Eøs? :) hvis den bare skal vises dersom det faktisk er en eøs søknad burde vi kanskje sjekke på erEøs i tillegg?